tpgtools/handwritten/utils.go

```diff
@@ -1,10 +1,13 @@
+// Contains functions that don't really belong anywhere else.
```

Comment:
Am I missing something, or is this file now managed by both mmv1+tpgtools? Should we delete the MMv1 one if so?

Reply:
Yes, good call! It gets overwritten anyways right now